lib/schedule/data.ex

```diff
@@ -378,6 +399,13 @@ defmodule Schedule.Data do
     |> Schedule.Helpers.merge_lists()
   end
 
+  @spec timepoint_names_for_ids(Timepoint.timepoints_by_id()) :: timepoint_names_by_id()
```

Comment:
We have both `timepoints_by_id` and `timepoint_names_by_id`, which are essentially the same thing, since timepoints are just an id and name. Do you think it'd be possible to consolidate them into one?

(It might not be that simple, in which case don't worry about it.)